js/bittrex.js

```diff
@@ -615,6 +615,9 @@ module.exports = class bittrex extends Exchange {
             }
             throw e;
         }
+        if (!response['result']) {
```

Comment:
Hi! What do they reply instead, when `result` is missing? Can you please post their verbose response for that case or tell how to reproduce it?

Reply:
```javascript
{ success: true, message: '', result: null }
```

To reproduce, just `fetchOrder` a non-existing order id like `3e35df1b-4f74-49ae-83ab-5224161edb11`.